tangelo/toolboxes/molecular_computation/rdms.py

```diff
@@ -197,3 +197,84 @@ def energy_from_rdms(ferm_op, one_rdm, two_rdm):
     e = core_constant + np.sum(one_electron_integrals * one_rdm) + np.sum(two_electron_integrals * two_rdm)
 
     return e.real
+
+
+def pad_rdms_with_frozen_orbitals(sec_mol, onerdm, twordm):
```

Comment:
There are almost no comments about what you do in the code blocks appearing in the body of the function.
If you're fine with that it's ok, if you can think of anything meaningful it is welcome.

Reply:
I added some comments, where it was more ciritical.